### DISCUSSION TITLE
Telegram Notify on Docker?

### DISCUSSION THREAD

**Discussion by Taavi95** [vogler/free-games-claimer#59] [Q&A] [answered]
Hey, thank you for the great script. I got it to work on my Linux machine and its running fine! Great Job and thank you for sharing this with us lazy people! :D 

I whish to get notified on telegram. But i didnt find out if that is possible in Docker. 
If it is possible, could you show me a example please? 

Thank you all in advance!

**Comment by vogler** [comment, marked as answer]
You just need to add `-e NOTIFY='tgram://...'` to your `docker run` command.
Also see #57.
